src/test/components/MenuButtons.test.js

```diff
@@ -518,6 +517,57 @@ describe('app/MenuButtons', function () {
       expect(getMetaInfoPanel()).toMatchSnapshot();
     });
 
+    it('with no extra info there is no more info button', async () => {
```

Comment:
I moved the tests to this file and made them a bit simpler and closer to the existing style.